<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>word_game</class>
 <widget class="QMainWindow" name="word_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>word_game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="showInformation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="showListButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;asdfasd&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>查看当前排行榜</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QComboBox" name="levelBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>level 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>level 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>level 3</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="searchLine">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查找</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ch">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>challenger</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="qu">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>93</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>quizmaster</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
